src/php/tests/interop/interop_client.php

```diff
@@ -36,6 +36,9 @@
 require 'message_set.php';
 require 'messages.php';
 require 'test.php';
+use Google\Auth\CredentialsLoader;
+use Google\Auth\ApplicationDefaultCredentials;
+use GuzzleHttp\ClientInterface;
```

Comment:
So that when I use the class I don't need to fully qualify the class with the namespace each time. Just to save a few characters on long lines